src/Token/TokenBase.tsx

```diff
@@ -48,16 +49,16 @@ export interface TokenBaseProps
    * Which size the token will be rendered at
    */
   size?: TokenSizeKeys
-}
 
-type TokenElements = HTMLSpanElement | HTMLButtonElement | HTMLAnchorElement
+  href?: string
```

Comment:
we need href defined, because `Token` uses it internally. and typescript will yell if we try to use this without adding it, since it's not, by default, allowed everywhere. 

adding it here allows its use across all of the TokenBase wrappers, which ensures it can work correctly